README.md

```diff
@@ -87,6 +87,73 @@ docker run -p 9101:9101 quay.io/prometheus/haproxy-exporter:v0.9.0 --haproxy.scr
 [hub]: https://hub.docker.com/r/prom/haproxy-exporter/
 [quay]: https://quay.io/repository/prometheus/haproxy-exporter
 
+### Available metrics
+
+| metric | description |
+| -------|:-----------:|
```

Comment:
Nit: I find this second column very hard to read due to the alignment. Please left align this as well.